coreos-devel/mantle/mantle-9999.ebuild

```diff
@@ -11,7 +11,7 @@ if [[ "${PV}" == 9999 ]]; then
 	COREOS_GO_MOD="vendor"
```

Comment:
I don't think the new version is going to build unless you move this variable to global scope (see the previous commit).

Reply:
whoops. you're right. fixed.